charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -157,3 +157,8 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - oidc-discovery.example.org
+
+tests:
+  hostAliases: []
+  tls:
+    customCA: ""
```

Comment:
Doubt a bit of exposing the test things on the public facing API.

Reply:
If the user is testing with their own CA (a lot of companies run their own private ones), this is the only way to ensure it works.